test-fixtures/suite1/main.smithy

```diff
@@ -10,15 +10,9 @@ service Weather {
 
 @readonly
 operation GetCurrentTime {
-    input: GetCurrentTimeInput,
-    output: GetCurrentTimeOutput
-}
-
-@input
-structure GetCurrentTimeInput {}
-
-@output
-structure GetCurrentTimeOutput {
-    @required
-    time: Timestamp
+    input := {},
```

Comment:
Because this is now a version 2 file, I think we should remove the trailing commas. Same in the other test suites.

Reply:
Removed in https://github.com/awslabs/smithy-vscode/pull/52/commits/9e434a07721b0f7e3791343e5e5b1448fcb13566.